mediactl/metadata_test.go

```
@@ -101,3 +101,39 @@ func TestMakeMetadataCoercesInvalidDBusStrings(t *testing.T) {
+			name:   "valid artURL unchanged",
+			artURL: "file:///tmp/cover.jpg",
+			want:   "file:///tmp/cover.jpg",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			track := playback.Track{ArtURL: tt.artURL}
+			got := makeMetadata(track, trackPath(1))
+
+			gotArtURL := got["mpris:artUrl"].Value().(string)
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)